app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -183,6 +185,27 @@ def test_clear_branches_deletes_branches_without_deleting_experiment(self):
         self.assertEqual(experiment.branches.count(), 0)
         self.assertEqual(experiment.changes.count(), 1)
 
+    @parameterized.expand(
```

Comment:
Hmm, this is probably a thing we should slap on more status-dependent tests for the sake of completeness

Reply:
Yeah I've been thinking about that, prolly the kind of thing we can do on a downtime maintenance week 👍